src/panel.css

```diff
@@ -221,7 +221,7 @@ a:hover {
   transition: var(--transition);
 }
 
-.remove-btn {
+.remove-btn, .allow-btn {
```

Comment:
All the CSS changes were extending the classes already defined for the remove panel.